packages/web/src/features/rental-business/components/SelectRangeCalendar/_atomic/RangeCalendar.tsx

```diff
@@ -18,6 +18,30 @@ const RangeCalendar: React.FC<RangeCalendarProps> = ({
   setReturnDate,
   workDates,
 }) => {
+  const [calendarSize, setCalendarSize] = useState<"sm" | "md" | "lg">("lg");
```

Comment:
`CalendarSizeProps`를 어딘가에 정의햇던 것 같은데 가져다 쓰면 어떨까 ㅇ.ㅇ
```suggestion
  const [calendarSize, setCalendarSize] = useState<CalendarSizeProps>("lg");
```

Reply:
이거 가져다 쓰니까 `Argument of type 'string' is not assignable to parameter of type 'CalendarSizeProps | (() => CalendarSizeProps)'.` 이러한 에러가 뜨는데 이게 calendarSize 가져다 쓰는 모든 것에 나타나서 안 쓰는게 더 깔끔할 것 같기도..?
따로 타입을 새로 정의해서 정리할까 했는데 뭔가 온갖 걸 바꿔야 할 것 같아서 일단은,,그대로 둘게용 ㅠ